freqtrade/freqtradebot.py

```diff
@@ -1305,7 +1306,7 @@ def exit_positions(self, trades: list[Trade]) -> int:
                     )
                 # Check if we can sell our current pair
                 if not trade.has_open_orders and trade.is_open and self.handle_trade(trade):
-                    trades_closed += 1
+                    trades_closed.append(trade)
```

Comment:
All this condition means is that an exit order was placed
there's a myrad of things that can happen to an order so it doesn't get filled.

There's absolutely no checks for this - so this again, introduces buggy behavior.